src/utils/api.js

```diff
@@ -37,5 +37,18 @@ const getTopics = () =>{
     })
 }
 
+const sendVoteByArticleId = (article_id) => {
+    const patchBody = {inc_votes: 1}
+    return axios.patch(`https://newsbase.onrender.com/api/articles/${article_id}`, patchBody).then((res) => {
+return res
+    })
+}
+const removeVoteByArticleId = (article_id) => {
+    const patchBody = {inc_votes: -1}
+    return axios.patch(`https://newsbase.onrender.com/api/articles/${article_id}`, patchBody).then((res) => {
+       
+return res
+    })
+}
```

Comment:
You've got two almost identical functions here - any thoughts on how you can better adhere to DRY and avoid repeating yourself?